src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -206,11 +202,21 @@ internal fun Dependency.resolvedVersion(): String? = when (this) {
   else -> throw GradleException("Unknown Dependency subtype: \n$this\n${javaClass.name}")
 }?.intern()
 
-private fun Dependency.isJavaPlatform(): Boolean = when (this) {
-  is ProjectDependency -> dependencyProject.pluginManager.hasPlugin("java-platform")
```

Comment:
This is not needed IMO. The **ProjectDependency** is also a **ModuleDependency** and it will have the attribute added by `testFixtures(...)` which is checked by the code below.